lib/extend/tag.js

```diff
@@ -236,7 +236,8 @@ class Tag {
 
     return Promise.fromCallback(cb => { this.env.renderString(str, options, cb); })
       .catch(err => Promise.reject(formatNunjucksError(err, str)))
-      .then(result => result.replace(rPlaceholder, (_, index) => cache[index]));
+      .then(result => result.replace(rPlaceholder, (_, index) => cache[index]))
+      .asCallback(callback);
```

Comment:
SInce `Promise.fromCallback` is used, is it still necessary to use `asCallback`?

https://github.com/hexojs/hexo/blob/0fd57368484886c9cb43ba7e44af4c75e0a08988/test/scripts/extend/tag.js#L167-L178

`callback` related test case is already added and it seems passed.